app.js

```diff
@@ -1,11 +1,21 @@
-// const appState = {
-//     userName,
-//     userSelectedItem,
-//     computerSelectedItem,
-//     round,
-//     points: [user, computer]
-// };
+ const appState = {
+    userName: '',
+    userSelectedItem: '',
+    computerSelectedItem: '',
+    round: 1,
+    points: [
+        {user: 0},
+        {computer: 0}
+    ]
+};
 
+
+// Modal getting elements
```

Comment:
in fact, variable `modal` is element itself. 
So - more proper comment would be:
`// Modal elements` (as we are not creating a function, but variable storing actual html element)